scripts/core/services/logger.js

```diff
@@ -30,15 +33,4 @@ export class Logger {
     }
 }
 
-/**
- * @ngdoc service
- * @module superdesk.core.services
- * @name loger
- *
- * @description
- * logger service
- *
- * Sends logs to sentry servers
- */
-export default angular.module('superdesk.core.services.logger', [])
-    .service('logger', ['config', Logger]);
```

Comment:
looks good, but I would keep the angular service where you can set logger as constant or something so it won't break any external apps which might be using it (like planning, publisher)